From: Matt Benjamin <mbenjamin@redhat.com>
Date: Wed, 17 Aug 2016 11:00:22 -0400
Subject: rgw_file:  log on write completion events at level 10

Signed-off-by: Matt Benjamin <mbenjamin@redhat.com>
(cherry picked from commit d1e512d8c0e20df5a96ccb5f775a7a2ca123efbe)

Resolves (with following): rhbz#1370331
---
 src/rgw/rgw_file.cc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/rgw/rgw_file.cc b/src/rgw/rgw_file.cc
index a8a2b3d..e58dd06 100644
--- a/src/rgw/rgw_file.cc
+++ b/src/rgw/rgw_file.cc
@@ -871,6 +871,10 @@ namespace rgw {
 
     file* f = get<file>(&variant_type);
     if (f && (f->write_req)) {
+      lsubdout(fs->get_context(), rgw, 10)
+	<< __func__
+	<< " finishing write trans on " << object_name()
+	<< dendl;
       rc = rgwlib.get_fe()->finish_req(f->write_req);
       if (! rc) {
 	rc = f->write_req->get_ret();
